python_modules/dagster/dagster_tests/core_tests/asset_defs_tests/test_assets_job.py

```diff
@@ -1304,11 +1304,11 @@ def test_subset_of_asset_job():
         assert materialization_events[1].asset_key == AssetKey("foo")
         assert materialization_events[2].asset_key == AssetKey("foo_bar")
 
-        with pytest.raises(DagsterInvalidSubsetError):
-            result = foo_job.execute_in_process(
-                instance=instance,
-                asset_selection=[AssetKey("unconnected")],
-            )
+        # with pytest.raises(DagsterInvalidSubsetError):
+        #     result = foo_job.execute_in_process(
+        #         instance=instance,
+        #         asset_selection=[AssetKey("unconnected")],
+        #     )
```

Comment:
im confused by why this was commented out for a logical version fix

Reply:
This is old so it's hard to remember, but piecing together from what I see, this was because the selection was no longer invalid due to the job definition change allowing for source assets to be included in an asset selection.

I can't recall why it wasn't simply deleted instead of commented out-- maybe just an oversight. Did this cause a problem or did you just find this commented out and wonder why?

Reply:
did not cause a specific problem, was just traversing blame to understand some things and came across it